### PR TITLE
Use less constants and fold the expressions into require statements

### PR DESCRIPTION
**Pull request by MrChico**
As discussed in https://github.com/axic/eth2-deposit-contract/pull/14#discussion_r424509800

### CODE REVIEW

**Comment by daejunpark** [comment]
This makes sense because the constants are already hardcoded in the deposit data root computation logic.

**Comment by axic** [comment]
Please rebase or merge in master.

**Review by axic — APPROVED** [review]

File: deposit_contract.sol [hunk 1]
```diff
@@ -87,12 +83,12 @@ contract DepositContract is IDepositContract, ERC165 {
...
-        require(signature.length == SIGNATURE_LENGTH, "DepositContract: invalid signature length");
+        require(pubkey.length == 48, "DepositContract: invalid pubkey length");
+        require(withdrawal_credentials.length == 32, "DepositContract: invalid withdrawal_credentials length");
+        require(signature.length == 96, "DepositContract: invalid signature length");
 
         // Check deposit amount
-        require(msg.value >= MIN_DEPOSIT_AMOUNT, "DepositContract: deposit value too low");
+        require(msg.value >= 1 ether, "DepositContract: deposit value too low");
         require(msg.value % GWEI == 0, "DepositContract: deposit value not multiple of gwei");
         uint deposit_amount = msg.value / GWEI;
```
Comment (axic) [inline deposit_contract.sol:93]: Should we change these to `1e9`? I think `GWEI` is fine. I wish `1 gwei` would work, but that will only come with Solidity 0.7.0.

Comment (MrChico) [reply deposit_contract.sol:93]: I think GWEI is more informative too

**Pull request merged**